setup.py

```diff
@@ -72,9 +73,7 @@
     ],
     description="Part 1 of Allen Cell and Structure Segmenter",
     entry_points={
-        "console_scripts": [
-            "my_example=aicssegmentation.bin.my_example:main"
-        ],
+        "console_scripts": ["my_example=aicssegmentation.bin.my_example:main"],
```

Comment:
i think this step should be  `batch_processing=aicssegmentation.bin.batch_processing:main`

here is another example: https://github.com/AllenCell/segmenter_model_zoo/blob/9cfe7e2882b29ec03409a9957eb6f27e90bbd3bc/setup.py#L72

Reply:
`my_example` is the dummy function in original cookie-cutter

Reply:
Good catch!